src/routes/EventsPage/index.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+
 import RichLayout from 'components/RichLayout'
-import { EventsFetcher } from 'endpoints'
 import Events from '../../components/Content/Events'
+import EventModel from 'endpoints/models/EventModel'
+import EVENTS_ENDPOINT from 'endpoints/events'
+import withFetcher from '../../endpoints/withFetcher'
+
+class ContentWrapper extends React.Component {
+  static propTypes = {
+    events: PropTypes.arrayOf(PropTypes.instanceOf(EventModel)).isRequired // From withFetcher
```

Comment:
Hmm... I kind of think, this comment is not necessary or not appropriate. The component shouldn't care where its properties come from imo, and it should be clear, that all properties (at least the required ones) need to be sent by the very parent component.

Reply:
Are we arguing here about too many comments? :P I think it's not always clear where properties come from so better document them. Maybe in a richer way though

Reply:
Yah, kind of arguing about too many comments 😆. Maybe we need some general commenting rules for the PropTypes. I see your point. Often we have HOCs which send us the properties, and then there are the properties a parent component needs to pass to its child itself. 🤔

Reply:
Will change to JavaDoc style comments